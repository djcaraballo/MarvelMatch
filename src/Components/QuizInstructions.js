import React from 'react'

const QuizInstructions = () => {
  return (
    <div>
      <h1>Take the quiz to find your <span>Spirit Superhero</span>!</h1>
      <p>We all know that no one in real life looks like a Marvel Comics superhero, but we all feel like one inside. Answer the questions in our brief quiz to choose your Spirit Superhero and avatar!</p>
 
      <p>P.S. Be honest with yourself! Your matches will be sent to you based on your superhero and there's no point in dating a Black Widow or Wolverine if you could date a Wasp or Luke Cage!</p>
    </div>
  )
}

export default QuizInstructions