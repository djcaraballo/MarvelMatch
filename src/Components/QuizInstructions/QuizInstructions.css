@import url('https://fonts.googleapis.com/css?family=Fjalla+One|Orbitron');

.directions-title {
  margin: 5rem auto 5rem;
}

.directions {
  font-size: 2.5em;
  line-height: 1.5em;
}

.directions-container {
  border: 1px #fff solid;
  margin: auto;
  width: 50%;
}

.quiz-nav {
  
}