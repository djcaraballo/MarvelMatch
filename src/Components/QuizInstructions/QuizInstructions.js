import React from 'react'
import { NavLink } from 'react-router-dom'
import './QuizInstructions.css'

const QuizInstructions = () => {
  return (
    <div>
      <h1 className="directions-title">Take the quiz to find your <span>Spirit Superhero</span>!</h1>
      <div className="directions-container">
        <p className="directions">We all know that no one in real life looks like a Marvel Comics superhero, but we all feel like one inside. Answer the questions in our brief quiz to choose your Spirit Superhero and avatar!</p>

      <NavLink
        to='/quiz'
        className='quiz-nav'>
        <h3>Match Me!</h3>
      </NavLink>
      </div>
    </div>
  )
}

export default QuizInstructions