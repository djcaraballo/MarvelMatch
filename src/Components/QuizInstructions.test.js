import React from 'react'
import QuizInstructions from './QuizInstructions'
import { shallow } from 'enzyme'

describe('QuizInstructions Component', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<QuizInstructions />)
  })

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })
})