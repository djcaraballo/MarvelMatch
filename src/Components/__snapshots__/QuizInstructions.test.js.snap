// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QuizInstructions Component should match the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <QuizInstructions />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1>
          Take the quiz to find your 
          <span>
            Spirit Superhero
          </span>
          !
        </h1>,
        <p>
          We all know that no one in real life looks like a Marvel Comics superhero, but we all feel like one inside. Answer the questions in our brief quiz to choose your Spirit Superhero and avatar!
        </p>,
        <p>
          P.S. Be honest with yourself! Your matches will be sent to you based on your superhero and there's no point in dating a Black Widow or Wolverine if you could date a Wasp or Luke Cage!
        </p>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "Take the quiz to find your ",
            <span>
              Spirit Superhero
            </span>,
            "!",
          ],
        },
        "ref": null,
        "rendered": Array [
          "Take the quiz to find your ",
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Spirit Superhero",
            },
            "ref": null,
            "rendered": "Spirit Superhero",
            "type": "span",
          },
          "!",
        ],
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "We all know that no one in real life looks like a Marvel Comics superhero, but we all feel like one inside. Answer the questions in our brief quiz to choose your Spirit Superhero and avatar!",
        },
        "ref": null,
        "rendered": "We all know that no one in real life looks like a Marvel Comics superhero, but we all feel like one inside. Answer the questions in our brief quiz to choose your Spirit Superhero and avatar!",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "P.S. Be honest with yourself! Your matches will be sent to you based on your superhero and there's no point in dating a Black Widow or Wolverine if you could date a Wasp or Luke Cage!",
        },
        "ref": null,
        "rendered": "P.S. Be honest with yourself! Your matches will be sent to you based on your superhero and there's no point in dating a Black Widow or Wolverine if you could date a Wasp or Luke Cage!",
        "type": "p",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1>
            Take the quiz to find your 
            <span>
              Spirit Superhero
            </span>
            !
          </h1>,
          <p>
            We all know that no one in real life looks like a Marvel Comics superhero, but we all feel like one inside. Answer the questions in our brief quiz to choose your Spirit Superhero and avatar!
          </p>,
          <p>
            P.S. Be honest with yourself! Your matches will be sent to you based on your superhero and there's no point in dating a Black Widow or Wolverine if you could date a Wasp or Luke Cage!
          </p>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "Take the quiz to find your ",
              <span>
                Spirit Superhero
              </span>,
              "!",
            ],
          },
          "ref": null,
          "rendered": Array [
            "Take the quiz to find your ",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Spirit Superhero",
              },
              "ref": null,
              "rendered": "Spirit Superhero",
              "type": "span",
            },
            "!",
          ],
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "We all know that no one in real life looks like a Marvel Comics superhero, but we all feel like one inside. Answer the questions in our brief quiz to choose your Spirit Superhero and avatar!",
          },
          "ref": null,
          "rendered": "We all know that no one in real life looks like a Marvel Comics superhero, but we all feel like one inside. Answer the questions in our brief quiz to choose your Spirit Superhero and avatar!",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "P.S. Be honest with yourself! Your matches will be sent to you based on your superhero and there's no point in dating a Black Widow or Wolverine if you could date a Wasp or Luke Cage!",
          },
          "ref": null,
          "rendered": "P.S. Be honest with yourself! Your matches will be sent to you based on your superhero and there's no point in dating a Black Widow or Wolverine if you could date a Wasp or Luke Cage!",
          "type": "p",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
